app/controllers/my_facilities_controller.rb

```diff
@@ -113,4 +92,13 @@ def force_cache?
   def report_with_exclusions?
     current_admin.feature_enabled?(:report_with_exclusions)
   end
+
+  def process_facility_stats(facilities, type)
+    stats_service = FacilityStatsService.new(accessible_facilities: @accessible_facilities, retain_facilities: facilities,
```

Comment:
What is the `retain_facilities` argument for here?  Is that really the `filtered_facilities` ?  Or maybe `selected_facilities` ?

Reply:
Yes, either filtered_facilities or selected_facilities would work. The idea of fetching general stats for all accessible facilities but returning specific data for only visible facilities was a change of scope that occurred on Thursday and I shoehorned it in a bit.